roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```
@@ -44,7 +45,7 @@
   loop: "{{ cr_names.stdout_lines }}"
 
 - name: |
-    TEST Use openstack observabilityclient to verify mariadb metrics are stored in prometheus
+    Use openstack observabilityclient to verify mariadb metrics are stored in prometheus
```

```suggestion
    TEST Use openstack observabilityclient to verify mariadb metrics are stored in prometheus
```

Revert this part of the change

---

Changed